### PR TITLE
Get the site ready for release

### PR DESCRIPTION
**Pull request by aabrook**
A small update that adds description, title, favicon, formatting.

screen shot 2016-11-15 at 4 11 43 pm

### CODE REVIEW

File: source/components/Preamble/__tests__/Preamble-test.js [hunk 1]
```diff
@@ -42,7 +42,7 @@ describe('Preamble display component', () => {
     const wrapper = mount(<Preamble images={images} score={-1} />)
 
     assert(wrapper.text() ===
-      'Thank you for donating with Everydayhero. You\'re awesome!' +
+      'Thank you for donating with everydayhero. You\'re awesome!' +
```
Comment (kizza) [inline source/components/Preamble/__tests__/Preamble-test.js]: :+1:

File: source/client.js [hunk 2]
```diff
@@ -1,6 +1,8 @@
 import React from 'react'
 import { render } from 'react-dom'
+import Helmet from 'react-helmet'
 import images from './images'
+import { links, metatags } from './lib/header'
```
Comment (bradparker) [inline source/client.js]: I maybe wouldn't do this in client...

You could do it in your main layout component, and then it'll run both server (during build) and client-side. But at the moment I don't think this'll be part of the server build ... which is not what you want.

File: source/lib/header.js [hunk 3]
```diff
@@ -0,0 +1,5 @@
+export const metatags = [
+  { 'name': 'description', 'content': 'Make your voice, time, effort and money all count for the causes you love and see your impact on your Giving Footprint with everydayhero. Get started now!' }
+]
+
+export const links = [{ 'rel': 'icon', 'href': `${process.env.BASE_PATH}/favicon.ico` }]
```
Comment (bradparker) [inline source/lib/header.js]: If this asset URL get's handled by webpack then you don't need to worry about BASE_PATH (I can recommend forgetting it's existence at every opportunity)

File: source/client.js [hunk 4]
```diff
@@ -13,7 +15,11 @@ const pageDetails = parse(window.location.search)
 
 render(
   <div>
```
Comment (bradparker) [inline source/client.js]: Oh I see... yeah, better to pull this out into a `Root` or `App` component. You can `renderToStaticMarkup` it during build. Will show tomorrow if you like?

Comment (aabrook) [inline source/client.js]: I was wondering when the line would arrive between App component and initial render. This was feeling like it should be a component but ... excuses.
I have refactored out <App />
Improving the meta tags would be nice so happy to chat tomorrow.

**Comment by aabrook** [comment]
Refactors are done. Would be good to know if there is a better approach to using /lib. It is now imported in a component so might make more sense, or pass it down?

File: source/client.js (hunk 4 above)
Comment (bradparker) [inline source/client.js]: @aabrook the component is good, but it's still only rendering on the client side.
Which is fine, but if you `renderToStaticMarkup(<App />)` in `renderDocument` on the server side, it'd be built into the resulting `index.html` file

Comment (aabrook) [inline source/client.js]: Ahhhh cool. Will check that out

Comment (aabrook) [inline source/client.js]: Is it possible to exclude libraries from the server side rendering? Mixpanel is not happy

Comment (aabrook) [inline source/client.js]: Mixpanel has dependencies on `document` existing and renderToStaticMarkup is hitting MP code and failing

Comment (bradparker) [inline source/client.js]: How does mix panel init?

Comment (aabrook) [inline source/client.js]: It checks for document.body, then sets a timeout if body cannot be found

Comment (bradparker) [inline source/client.js]: When it's required? Or when you make a new instance of the client?

Comment (bradparker) [inline source/client.js]: If it's during `require` then things are though, and they're being a bit of a shit.
If it's only when you `new Mixpanel()` then it's easy `typeof document !== 'undfined' && new Mixpanel()` or likewise

Comment (bradparker) [inline source/client.js]: This looks promising:
https://github.com/mixpanel/mixpanel-js/blob/master/build/mixpanel.cjs.js#L8-L16

Comment (aabrook) [inline source/client.js]: Must be on import then. When I have `if(!document)` or `if(document !== undefined)` before init it fails

Comment (aabrook) [inline source/client.js]: This is the problem line of code
https://github.com/mixpanel/mixpanel-js/blob/master/build/mixpanel.cjs.js#L2140

Comment (bradparker) [inline source/client.js]: This whole thing is pretty gross...

This looks like a problem.
https://github.com/mixpanel/mixpanel-js/blob/master/build/mixpanel.cjs.js#L5478

Comment (bradparker) [inline source/client.js]: Kinda makes the care displayed here:
https://github.com/mixpanel/mixpanel-js/blob/master/build/mixpanel.cjs.js#L33
Pointless

Comment (aabrook) [inline source/client.js]: Yeah when I dove through MP when I added it to command-centre, gross was a pretty apt description

Comment (bradparker) [inline source/client.js]: There are a few options then.

1) Abandon trying to do this... render only client side, move on
2) Put the include of this piece of crap into client, but DI it into your App perhaps on a prop?

```
// client.js
import mixPanel from 'mixpanel'

import App from './App'

render(<App  analyticsClient={mixPanel} />)

// server.js

import App from './App'

renderDocument({
  content: renderToStaticMarkup(<App />)
})

// App.js

const ({
  analtyicsClient = defaultImplemenation // so things will still work
})
```

Comment (aabrook) [inline source/client.js]: I don't like the current default so I think rendering an empty div to start? Not very progressive though ... I'll think about it while I work on the email template across email clients

Comment (bradparker) [inline source/client.js]: my `<div />` was for brevity. I can make this change if you like.

Comment (aabrook) [inline source/client.js]: If you could please that'd be awesome.

File: source/server.js [hunk 5]
```diff
@@ -32,6 +37,7 @@ export default ({ assets }) => (route) => (
   Promise.resolve({
     result: renderDocument({
       content: renderToString(<App />),
+      head: Helmet.rewind(),
```
Comment (aabrook) [inline source/server.js]: Big :+1:

**Comment by bradparker** [comment]
@kizza mergy mergy mergy?

**Pull request merged**